go.mod

```
@@ -24,19 +24,20 @@ require (
 	go.etcd.io/etcd v0.5.0-alpha.5.0.20200910180754-dd1b699fc489
 	golang.org/x/sys v0.0.0-20210225134936-a50acf3fe073
 	google.golang.org/grpc v1.29.1
-	k8s.io/api v0.21.0
-	k8s.io/apiextensions-apiserver v0.21.0
-	k8s.io/apimachinery v0.21.0
-	k8s.io/apiserver v0.21.0
-	k8s.io/client-go v0.21.0
-	k8s.io/component-base v0.21.0
+	k8s.io/api v0.21.1
+	k8s.io/apiextensions-apiserver v0.21.1
+	k8s.io/apimachinery v0.21.1
+	k8s.io/apiserver v0.21.1
+	k8s.io/client-go v0.21.1
+	k8s.io/component-base v0.21.1
 	k8s.io/cri-api v0.21.0
 	k8s.io/klog/v2 v2.8.0
 	k8s.io/utils v0.0.0-20201110183641-67b214c5f920
 )
 
 replace (
 	github.com/gogo/protobuf => github.com/gogo/protobuf v1.3.2
+	github.com/openshift/library-go => /Users/lszaszki/go/src/github.com/openshift/library-go
```

Was this leftover are you waiting for a PR to merge?

---

yes, it requires https://github.com/openshift/library-go/pull/1098

---

Asking as the CI is failing due to this, as there is no such file. Should we put this on hold until that is merged maybe?

> ERRO[2021-06-07T15:26:37Z] go: github.com/openshift/library-go@v0.0.0-20210414082648-6e767630a0dc (replaced by /Users/lszaszki/go/src/github.com/openshift/library-go): reading /Users/lszaszki/go/src/github.com/openshift/library-go/go.mod: open /Users/lszaszki/go/src/github.com/openshift/library-go/go.mod: no such file or directory 

Otherwise we can do this maybe:
```suggestion
	github.com/openshift/library-go => github.com/lszaszki/library-go
```

---

only `ci/prow/verify-deps`. I put a hold on that PR, once 1098 merges I will change this PR to link library-go instead of my local repository.

---

done